homeassistant/components/yamaha/media_player.py

```diff
@@ -368,10 +366,27 @@ def play_media(self, media_type, media_id, **kwargs):
         if media_type == "NET RADIO":
             self.receiver.net_radio(media_id)
 
+    async def async_enable_output(self, enabled, port):
```

Comment:
We don't need these async wrappers. Our service helper will handle sync methods too.